### PR TITLE
Feat/owrt stqa 73 add owner comment

### PR DESCRIPTION
**Pull request by 1831239**
I finished this user story.

### CODE REVIEW

File: src/main/resources/db/postgresql/populateDB.sql [hunk 1]
```diff
@@ -22,16 +22,16 @@ INSERT INTO types VALUES (4, 'snake') ON CONFLICT DO NOTHING;
...
+INSERT INTO owners VALUES (5, 'Peter', 'McTavish', '2387 S. Fair Way', 'Madison', '6085552765', 'peter.mctavish@gmail.com', 'This owner is always a bit late') ON CONFLICT DO NOTHING;
+INSERT INTO owners VALUES (6, 'Jean', 'Coleman', '105 N. Lake St.', 'Monona', '6085552654', 'jean.coleman@gmail.com', 'This owner is very kind') ON CONFLICT DO NOTHING;
+INSERT INTO owners VALUES (7, 'Jeff', 'Black', '1450 Oak Blvd.', 'Monona', '6085555387', 'jeff.black@gmail.com', 'This owner likes pink') ON CONFLICT DO NOTHING;
+INSERT INTO owners VALUES (8, 'Maria', 'Escobito', '345 Maple St.', 'Madison', '6085557683', 'maria.escobito@gmail.com.com', 'This owner has peanut allergy') ON CONFLICT DO NOTHING;
+INSERT INTO owners VALUES (9, 'David', 'Schroeder', '2749 Blackhawk Trail', 'Madison', '6085559435', 'david.schroeder@gmail.com', 'This owner is new to this clinic') ON CONFLICT DO NOTHING;
+INSERT INTO owners VALUES (10, 'Carlos', 'Estaban', '2335 Independence La.', 'Waunakee', '6085555487', 'carlos.estaban@gmail.com', 'This owner hates animals') ON CONFLICT DO NOTHING;
```
Comment (s00220021) [inline src/main/resources/db/postgresql/populateDB.sql:34]: funny comments haha

File: src/main/java/org/springframework/samples/petclinic/model/Owner.java [hunk 2]
```diff
@@ -59,6 +59,10 @@
     @NotEmpty
     private String email;
 
+    @Column(name = "comment")
+    @NotEmpty
+    private String comment;
+
```
Comment (s00220021) [inline src/main/java/org/springframework/samples/petclinic/model/Owner.java:65]: maybe u can add a comment to say the reason for initializing

File: src/test/java/org/springframework/samples/petclinic/web/OwnerControllerTests.java [hunk 3]
```diff
@@ -20,6 +20,8 @@
 /**
  * Test class for {@link OwnerController}
  *
+ *
+ *
```
Comment (s00220021) [inline src/test/java/org/springframework/samples/petclinic/web/OwnerControllerTests.java:24]: good tests, keep it up!

**Review by s00220021 — DISMISSED** [review]
Overall very good. Maybe adding comments on the changes u did would help to see what was done. Good job bro!

**Review by lucas-cimino — DISMISSED** [review]
Good job Illia, everything from the comment field you implemented seems right!

**Review by s00220021 — APPROVED** [review]

**Review by lucas-cimino — APPROVED** [review]
You found the error in the AbstractClinicServiceTests.java test class, good job!

**Pull request merged**